### ISSUE TITLE
Hello, have you forgotten this project...?

### ISSUE THREAD

**Issue by Yuanhf3754** [Manojbhat09/Trajformer#3]
Look forward to the code release...

**Comment by fgqile** [comment]
yes,i also want to see the code. i am working at the vehicle driveing.

**Comment by Manojbhat09** [comment]
Hello,
Apologies for the delay, 
We are in between legal process of two companies before publishing the codebase to open-source.
Please pose your doubts in here, we are happy to help.
